python/ray/data/tests/test_stats.py

```diff
@@ -471,6 +477,94 @@ def test_dataset_stats_basic(ray_start_regular_shared, enable_auto_log_stats):
             )
 
 
+def test_dataset_hide_verbose_log(ray_start_regular_shared, enable_auto_log_stats):
+    context = DataContext.get_current()
+    context.optimize_fuse_stages = True
+    context.verbose_stats_logs = False
+
+    if context.new_execution_backend:
```

Comment:
i recently removed these context attributes (new_execution_backend, use_streaming_executor) -- you should treat them as `True` in all cases, which should help simplify the test